client/verta/verta/environment/_environment.py

```diff
@@ -19,7 +19,8 @@ class _Environment(_blob.Blob):
     Handles environment variables and command line arguments.
 
     """
-    def __init__(self, env_vars, autocapture):
+
+    def __init__(self, env_vars, autocapture, apt_packages=None):
```

Comment:
I'd wish to keep `autocapture` as the last param because it feels more right to me to group `env_vars` and `apt_packages` side-by-side, and this is an internally-used class, but this is 👌 to keep the change manageable.

Reply:
I believe that parameters with default arguments need to come after those without.